### PR TITLE
Complete the 'Ethereum Dapp Develope Tutorial' section one

### PR DESCRIPTION
**Pull request by RainyDay1999**
Section one:
The prior knowledge of tutorial.

### CODE REVIEW

**Comment by vsszhang** [comment]
Add more thing, submission is too small.

File: tutorial/Ethereum Dapp Develope [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+Before we develop a ture DApp on Ethereum. Let us see what we need to know first.
+
+So, here is a list about you must know of Ethereum ecosystem.
+- How to write Ethereum smart contract (Solidity)?
+- How to call contract?
+  - What SDK can I use?
+  - How can I check the contract calling condition (Blockchain browser).
+- How can I operate with the original token, NF or NFT?
+- Are there any common wallet I can use?
```
Comment (vsszhang) [inline tutorial/Ethereum Dapp Develope:11]: Add some wallet example

File: tutorial/Ethereum Dapp Develope [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# How to develope a Ethereum DApp
```
Comment (vsszhang) [inline tutorial/Ethereum Dapp Develope:1]: submission is too small, add more thing

**Review by vsszhang — CHANGES_REQUESTED** [review]
Section one review

File: tutorial/Ethereum Dapp Develope (hunk 1 above)
Comment (RainyDay1999) [reply tutorial/Ethereum Dapp Develope:11]: added

File: tutorial/Ethereum Dapp Develope (hunk 2 above)
Comment (RainyDay1999) [reply tutorial/Ethereum Dapp Develope:1]: view common wallet desc commit.

**Comment by RainyDay1999** [comment]
reopen it, maybe

**Pull request merged**